src/utils/availability.js

```diff
@@ -1,9 +1,23 @@
 
 let availabilityChannel = undefined;
-const getAvailabilityChannel = async () => { return availabilityChannel }
-const setAvailabilityChannel = (channel) => { availabilityChannel = channel }
+const getAvailabilityChannel = async () => { return availabilityChannel; };
+const setAvailabilityChannel = channel => { availabilityChannel = channel; };
 
+const availability = async newChannel => {
```

Comment:
Can this method name be changed to something more understandable at glance? Maybe `updateAvailabilityChannel`?